packages/esm-patient-chart-app/translations/en.json

```diff
@@ -71,6 +72,11 @@
   "noObservationsFound": "No observations found",
   "notes": "Notes",
   "openAnyway": "Open anyway",
+  "openFormSpecificWarningText": "The {workspaceTitle} form will not be saved if you exit the patient chart now.",
```

Comment:
Will the form be saved if i exit the patient chart `later` instead of `now`? 😊

Reply:
If later comes after the form is saved, I can change that but the [zeplin mocks](https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/61bb8ad37c91686a70eec2d2) have the wording that way. You might suggest a better word @dkayiwa

Reply:
I would also tell the user what they need to do, to have the form saved. 😊

Reply:
That sounds much better, thanks, Daniel.